convert2rhel/checks.py

```diff
@@ -370,14 +370,24 @@ def ensure_compatibility_of_kmods():
     if not unsupported_kmods:
         logger.debug("All loaded kernel modules are available in RHEL.")
     else:
-        logger.critical(
-            "The following loaded kernel modules are not available in RHEL:\n{0}\n"
-            "First, make sure you have updated the kernel to the latest available version and rebooted the system.\n"
-            "If this message appears again after doing the above, prevent the modules from loading by following {1}"
-            " and run convert2rhel again to continue with the conversion.".format(
-                "\n".join(unsupported_kmods), LINK_PREVENT_KMODS_FROM_LOADING
+
+        if "CONVERT2RHEL_UNSUPPORTED_UNCHECKED_KMODS" in os.environ:
+            logger.warning(
+                "Detected 'CONVERT2RHEL_UNSUPPORTED_UNCHECKED_KMODS' environment variable."
+                " We will continue the conversion, with the following kernel modules that are not supported:\n{kmods}\n".format(
+                    kmods=not_supported_kmods
+                )
+            )
+
+        else:
+            logger.critical(
+                "The following loaded kernel modules are not available in RHEL:\n{0}\n"
+                "First, make sure you have updated the kernel to the latest available version and rebooted the system.\n"
+                "If this message appears again after doing the above, prevent the modules from loading by following {1}"
+                " and run convert2rhel again to continue with the conversion.".format(
+                    "\n".join(not_supported_kmods), LINK_PREVENT_KMODS_FROM_LOADING
+                )
```

Comment:
```suggestion
        unsupported_kmods = get_unsupported_kmods(host_kmods, rhel_supported_kmods)
      
        if "CONVERT2RHEL_UNSUPPORTED_UNCHECKED_KMODS" in os.environ:
            logger.warning(
                "Detected 'CONVERT2RHEL_UNSUPPORTED_UNCHECKED_KMODS' environment variable."
                " We will continue the conversion, with the following kernel modules that are not supported:\n{kmods}\n".format(
                    kmods="\n".join(unsupported_kmods)
                )
            )

        else:
            logger.critical(
                "The following loaded kernel modules are not available in RHEL:\n{0}\n"
                "First, make sure you have updated the kernel to the latest available version and rebooted the system.\n"
                "If this message appears again after doing the above, prevent the modules from loading by following {1}"
                " and run convert2rhel again to continue with the conversion.".format(
                    "\n".join(unsupported_kmods), LINK_PREVENT_KMODS_FROM_LOADING
                )
```

Reply:
@Andrew-ang9, you removed too much in your last commit. The proper thing here would be this, just like it is in upstream.